templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+  <p>Are you sure to delete the manufacturer: {{ manufacturer }}</p>
+  <p>All cars with this manufacturer will be also deleted</p>
+  <form action="" method="post" novalidate>
+  {% csrf_token %}
+  <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.